.github/workflows/full_stack_tests.yml

```diff
@@ -100,11 +100,13 @@ jobs:
           SUITE_NAME=${{ matrix.suite.name }}
           echo "MODIFIED_SUITE_NAME=${SUITE_NAME//\//_}" >> $GITHUB_OUTPUT
       - name: Run E2E Test ${{ matrix.suite.name }}
-        run: >
-          VIDEO_RECORDING_IS_ENABLED=0
-          xvfb-run -a --server-args="-screen 0, 1285x1000x24"
-          python -m scripts.run_e2e_tests --skip-install
-          --skip-build --suite=${{ matrix.suite.name }} --prod_env --server_log_level=info | tee e2e_test_${{ steps.generate_suite_name_for_artifacts.outputs.MODIFIED_SUITE_NAME }}.log
+        run: |
```

Comment:
Is there a reason you changed this from `>` to `|`? We generally prefer to avoid trailing `\` since it can cause issues if there is a space after it.

Maybe use a semicolon to separate the commands?

Reply:
I thought `|` to be more readable. I have updated workflow file to use `>` again.

Reply:
Yes, we can use semicolon to seperate the commands. I have updated workflow file to use `>` and separate commands using `;`